.github/workflows/Pkg EdFi.Ods.Extensions.Homograph.yml

```
@@ -27,13 +27,19 @@ env:
   HEAD_REF:  ${{ GITHUB.HEAD_REF }}
   REF_NAME:  ${{ GITHUB.REF_NAME }}
   REPOSITORY_DISPATCH_BRANCH: ${{ github.event.client_payload.branch }}
+  REPOSITORY_OWNER: ${{ GITHUB.REPOSITORY_OWNER }}
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     steps:
+    - name: Check for Azure token
+      if: ${{ env.REPOSITORY_OWNER == 'Ed-Fi-Alliance-OSS' && env.AZURE_ARTIFACT_NUGET_KEY == '' }}
+      run: | 
+        echo "Missing Azure Token" 
```

You may want to consider:

```suggestion
        echo "::error::Missing Azure Token" 
```

to create more visibility via an annotated message. https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions